front/src/components/scene/FormulaFunctionsHelp.css

```
@@ -0,0 +1,14 @@
+   user only scans when writing a formula, it must not compete with the input itself. */
+.formulaHelp {
+  font-size: 11px;
+  line-height: 1.5;
+  opacity: .75;
+  margin-bottom: .375rem;
+}
+
+.formulaHelp code {
+  background: none;
+  padding: 0;
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)